src/server/stream_family.cc

```diff
@@ -836,6 +1005,11 @@ OpResult<vector<GroupInfo>> OpListGroups(const DbContext& db_cntx, string_view k
       ginfo.consumer_size = raxSize(cg->consumers);
       ginfo.pending_size = raxSize(cg->pel);
       ginfo.last_id = cg->last_id;
+      // ginfo.entries_read = cg->entries_read;  // TODO : this returns incorrect value, check.
```

Comment:
`cg->entries_read` returns entries_read as 0 after the messages are read using xreadgroup.
I have traced the implementation of xread and xreadgroup but can't see entries_read being updated anywhere.
@kostasrim Can you help confirm this ?

Reply:
Debugged the xread command and noticed we do not update the `entries_read` field.
`OpRead` function makes use of `OpRangeFromConsumerPEL` & `OpRange`. Both should not modify the `entries_read` field for a group.
It would make sense to update `entries_read` after these calls.
What do you think ?
Also, it would make sense to fix this in a separate PR.

Reply:
Totally. Please try making this PR as small and self contained as possible even if we miss some semantics. This change is already big as it is. If you find more such issues - you can just add a `TODO: ` comment or open an issue in the repo

Reply:
Created an issue https://github.com/dragonflydb/dragonfly/issues/1945 to address this bug